plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -349,8 +349,9 @@ newtype CekM uni fun s a = CekM
     } deriving newtype (Functor, Applicative, Monad)
 
 -- | The CEK machine-specific 'EvaluationException'.
-type CekEvaluationException uni fun =
-    EvaluationException CekUserError (MachineError fun) (Term Name uni fun ())
+-- TODO: unparameterize the `name` when we move to direct-debruijn api
```

Comment:
do it

Reply:
ah no, it's an old todo where I thought that we will stop using wrappers, like Cek.hs module does. I will remove the todo.